src/index.html

```diff
@@ -11,7 +11,183 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="personalInfo chapter" name="personalInfo">
+        <legend>Personal information</legend>
+
+        <div class="block">
+          <label for="surname">
+            Surname:
+            <input type="text" name="surname" id="surname">
+          </label>
+        </div>
+
+        <div class="block">
+          <label for="name">
+            Name:
+            <input type="text" name="name" id="name">
+          </label>
+        </div>
+
+        <div class="block">
+          <label for="age">
+            How old are You?
+            <input
+            type="number"
+            name="age"
+            id="age"
+            min="1"
+            max="100"
+            value="12"
+            >
```

Comment:
add correct intend for all arguments
```suggestion
            <input
              type="number"
              name="age"
              id="age"
              min="1"
              max="100"
              value="12"
            >
```